scripts/filter.js

```diff
@@ -0,0 +1,321 @@
+var defaultDiacriticsRemovalap = [
```

Comment:
This code is from here https://stackoverflow.com/questions/30844353/how-to-remove-diacritics-accents-from-a-string